packages/react-core/src/components/CalendarMonth/examples/CalendarMonth.md

```diff
@@ -17,3 +17,12 @@ beta: true
 
 ```ts file="./CalendarMonthSelectableDate.tsx"
 ```
+
+### Date range
+
+In this example, there are two dates selected: the first is a range start date (via the `rangeStart` prop) and the second is a range end date (via the `date` prop). For the purposes of this example, these dates are static and cannot be updated.
```

Comment:
@wolfeallison do you mind taking a quick look at the copy here.  We are trying to get  across the point that this is a static example and you can not change the date range.  We have a link to a deme for the date picker where we show how to chose a date range.

Reply:
Hi! I slightly changed some of it, like using a numeral instead of a number and made it a little more concise. "In this example, there are 2 dates selected: a range start date (via the `rangeStart` prop) and a range end date (via the `date` prop). For this example, these dates are static and cannot be updated. Additionally, any dates prior to the range start date are disabled by passing in an array of functions to the `validators` prop. In this case a single function is passed in, which checks whether a date is greater than or equal to the range start date."

Reply:
Because bold is used for component names or to help with grouping, I think it may be confusing to bold other parts. What if we put "For this example, these dates are static and cannot be updated" at the start of a new paragraph so the user is more likely to read and see it?

Reply:
Based on the feedback I've updated the verbiage per @wolfeallison comments. I also included a line to link to the date picker demos (which I can either further update or remove if it ends up not working in the context).